src/components/news-tab.yml

```diff
@@ -0,0 +1,33 @@
+latestActivitiesHeader:
+  en: Latest activities
+  de: Letzte Aktivitäten
+documentCreatedActivity:
+  en: New document created
+  de: Neues Dokument erstellt
+documentUpdatedActivity:
+  en: Document updated
+  de: Dokument bearbeitet
+documentMarkedFavoriteActivity:
+  en: Document marked as favorite
+  de: Dokument als Favorit hinzugefügt
+roomCreatedActivity:
+  en: New room created
+  de: Neuen Raum erstellt
+roomUpdatedActivity:
+  en: Room updated
+  de: Raum bearbeitet
+roomMarkedFavoriteActivity:
+  en: Room marked as favorite
+  de: Raum als Favorit hinzugefügt
+roomJoinedActivity:
+  en: Private room joined
+  de: Privater Raum eingetreten
```

Comment:
Either "Privatem Raum beigetreten" or "Privaten Raum betreten" or "In privaten Raum eingetreten", there's a lot of choices :-) (The first one sounds fanciest)

Reply:
Thanks, fancy it is then! But mostly because it's the only one that starts with the entity type ("private room", like the others do) **and** is in Partizip 2 form.

Reply:
They all are Partizip 2 btw., but I liked the first one best too.